src/main/java/com/monumental/util/string/StringHelper.java

```diff
@@ -43,4 +43,197 @@ public static String removeBeginningAndEndingQuotes(String string) {
 
         return string;
     }
+
+    /**
+     * Method to parse a state abbreviation or full name into a common format
+     * @param state - State String to parse into common format
+     * @return String - The name of the State in a common format
+     */
+    public static String parseState(String state) {
+        if (state == null) {
+            return null;
+        }
+
+        switch (state.toLowerCase()) {
```

Comment:
I can't tell if I hate this for having 100 cases or like it for being simple and not requiring a dependency

Reply:
I figured it was easier to roll our own 🤷‍♂